incubator-tools/pdf_splitter/pdf_splitter.ipynb

```diff
@@ -0,0 +1,263 @@
+{
```

Comment:
<p>I don't think this tool is needed anymore. Document AI Toolbox can perform PDF Splitting</p><p> </p><p><a href="https://cloud.google.com/document-ai/docs/toolbox#pdf-split" target="_blank">https://cloud.google.com/document-ai/docs/toolbox#pdf-split</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
@holtskinner Thanks for pointing us to existing package, We observed both serves different purposess, `split_pdf` from docai-toolbox expects CDC/CDS json results to split pdf into chunks based on `document proto entity.type_  start_page & end_page` and again CDC/CDS processors has their own page limits to process pdf document, but this `pdf_splitter` tools expects only pdf and split into small chunks based on `CHUNK_SIZE` variable, so that user can use chunked pdfs against any processor. (for eg **260** `pages pdf ` will be chunked into **6** `pdfs` if `CHUNK_SIZE = 50`)